### ISSUE TITLE
Input element isn't honouring `w` property

### ISSUE THREAD

**Issue by yan5** [yesvelte/yesvelte#471]
Input element isn't honouring `w` property. Below doesn't bring the size to half of the parent div.
```
	<Input w="50" type="text" placeholder="" bind:value={email}
		><span slot="start"><Icon size="xs" name="at" /></span></Input
	>

```
![image](https://github.com/yesvelte/yesvelte/assets/20642737/d155e23b-6927-43a0-bb59-ec193a2f7291)

But below (wrapping it on `El`) will.
```
<El w="50">
	<Input type="text" placeholder="" bind:value={email}
		><span slot="start"><Icon size="xs" name="at" /></span></Input
	>
</El>
```
![image](https://github.com/yesvelte/yesvelte/assets/20642737/416572e6-2987-4332-97bc-ffa798c614e0)

**Comment by TheHadiAhmadi** [comment]
this is a bug and We'll fix this problem.

@yan5 our preferred way is to use `FormInput` instead of `Input` in these situations, you can set `col=6` instead of `w=50`. 

Here is a repl: https://svelte.dev/repl/ab1377eea17647c49b3aa8743963b420?version=3.55.1
Documentation link: https://www.yesvelte.com/docs/form/form-input#Layout

let me know if it solves your problem.

**Comment by yan5** [comment]
@TheHadiAhmadi thanks,

I was using FormInput, but couldn't find a way to show the icon along with input.. like '@' icon above. Any way to do it with FormInput?

**Comment by TheHadiAhmadi** [comment]
> I was using FormInput, but couldn't find a way to show the icon along with input.. like '@' icon above. Any way to do it with FormInput?

I've fixed the problem of FormInput in #485. FormInput has `start` and `end` slots similar to Input component.
